__tests__/gameState.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, test } from '@jest/globals';
+import { ethers } from 'ethers';
+import { SignedGameMove } from 'types/arbiter';
+import { testGameState0 as testGameState, TEST_ADDRESS_1, testGameMove1 } from '__fixtures__/testGame';
+
+const localStorage = window.localStorage;
+const privateStore = `${TEST_ADDRESS_1}_private`;
+
+describe('game state', () => {
+  test('no session private key in localStorage', async () => {
+    const privateKey = localStorage.getItem(privateStore);
+    expect(privateKey).toBeNull();
+    expect(async () => await testGameState.signMove(testGameMove1, TEST_ADDRESS_1)).rejects.toThrowError('Can not get wallet: no private key');
```

Comment:
"Can not get wallet : no key" is still on the generic error message side. Viewing such a message, to understand things like what the key is required for or which part of the codebase, one has to rely on stacktraces and such. What if aproper stack trace is inaccessible ? Dev who knows the codebase has to rely on this message being unique.

A better msg would say smth like

GameMoveSignerModule: can't sign move - no pk in local storage